src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/AssignRequest.cs

```diff
@@ -1,6 +1,24 @@
-﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerCommitmentsV2.Web.Extensions;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
-    public class AssignRequest : ConfirmProviderRequest
+    public class AssignRequest : BaseAssignRequest
     {
+        public long AccountLegalEntityId { get; set; }
+    }
+
+    public class BaseAssignRequest : IndexRequest
+    {
+        public long ProviderId { get; set; }
```

Comment:
I think `ProviderId` should be hidden from the URL as well, shouldn't it?

Reply:
provider id is entered by the user i the journey